handler/ezBIDS_core/ezBIDS_core.py

```diff
@@ -1374,7 +1374,7 @@ def modify_objects_info(dataset_list):
             protocol["headers"] = str(image.header).splitlines()[1:]
 
             object_img_array = image.dataobj
-            if object_img_array.dtype not in ["<i2", "<u2"]:
+            if object_img_array.dtype not in ["<i2", "<u2", "<f4"]:
```

Comment:
@dlevitas PET images are scaled so this is the best I could do for this.

Reply:
That should be fine. This if statement is really geared towards to MRI data, so if it's still catching PET data we could have it trigger only if the image's modality is MRI specifically.

Reply:
K, I added a PET check so it just accepts whatever nibabel puts out if modality is PT. If nibabel reads the nifti, I'm pretty satsified it's a valid PET image.